cache/variables.tf

```diff
@@ -9,6 +9,12 @@ variable "cache_bucket_prefix" {
   default     = ""
 }
 
+variable "cache_bucket_name_include_account_id" {
+  description = "Boolean to add current account ID to cache bucket name."
+  type        = string
```

Comment:
Please update the type to `bool` (without quotes). Next you can remove the quotes form the default value as well.

Reply:
Changed to bool